sdk/src/main/java/com/podio/sdk/domain/Reference.java

```diff
@@ -22,5 +23,9 @@ public long getId() {
     public String getTitle() {
         return title;
     }
+
+    public String getLink() {
```

Comment:
It's used to launch a specific reference (workspace or item) that is attached to a task in the podio app. 
![screen shot 2016-08-04 at 9 28 58 am](https://cloud.githubusercontent.com/assets/19800711/17403490/02a967ca-5a26-11e6-8cab-b8c7ca7aeec5.png)